src/examples/ui/sticky-header/css/index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+		<title>Sticky Header</title>
+
+		<link rel="stylesheet" href="@styles/base.css">
+		<link rel="stylesheet" href="index.css">
+
+	</head>
+    <body>
+
+		<div class="site" id="top">
+
+			<a class="button skip-link skip-link--to-content screen-reader-text" href="#content">Skip to content</a>
+
+			<header class="container site-header site-header--sticky">
+				<div class="site-header__inner">
+
+					<h1 class="header-logo">
+						<a href="./">Logo</a>
+					</h1>
+					<nav class="header-menu">
+						<h1>
+							Menu
+						</h1>
+						<ul>
+							<li><a href="#item-1">Item 1</a></li>
+							<li><a href="#item-2">Item 2</a></li>
+							<li><a href="#item-3">Item 3</a></li>
+							<li><a href="#item-4">Item 4</a></li>
+						</ul>
+					</nav>
+
+				</div>
+			</header>
+
+			<main class="site-content" id="content">
+				<div class="container site-content__inner">
+
+					<section class="container hero">
+						<h1>
+							Nullam id dolor id nibh ultricies vehicula.
+						</h1>
+						<p>
+							Praesent commodo cursus magna, vel consectetur et. <a href="#content-menu">Skip to On this page</a> | <a href="#item-2">Skip to Item 2</a>
+						</p>
+					</section>
```

Comment:
This is an example of where local hash link could exist. Its a common pattern for a page to have a page header, imo, it shows a variation for how a sticky header may visually interact with content

Reply:
Let's make sure to note that the hero is being used as a threshold somewhere so it clearer that's part of the purpose of the element in this case.

It didn't always seem to hide when I was testing at the edge of the hero, so that may need follow up.

Reply:
The hero is _not_ being used as a threshold but as and example of page content, color and layout variation for when the sticky header is positioned sticky over it.

The threshold is defined by the variable `scrollPositionThreshold` and function `calculateScrollPositionThreshold`. Currently the value is 2 times the `height` of the `.site-header`, which at desktop computes to `160px`. The hero height is greater than that so if you are scrolling slow enough you should see the bottom of the hero before the sticky header appears a that scroll position.

All of the demo content is in service of providing probable content scenarios where a hash link or anchor target may appear in content.